hci.go

```diff
@@ -0,0 +1,536 @@
+//go:build nina || nano_rp2040
+
+package bluetooth
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"errors"
+	"machine"
+	"time"
+)
+
+const (
```

Comment:
These constants also look like implementation details, and not something that needs to be part of the public API.

Reply:
changed to not export them.